docs/mdbook/src/android/location-component-guide.md

```diff
@@ -0,0 +1,101 @@
+# LocationComponent
+
+This guide will demonstrate how to utilize the [LocationComponent] to represent the user's current location.
+
+
+1. When implementing the [LocationComponent], the application should request location permissions.
+
+   Declare the need for foreground location in the `AndroidManifest.xml` file.
+
+   For more information, please refer to the [Android Developer Documentation].
+
+```xml
+<manifest ... >
+  <!-- Always include this permission -->
+  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+
+  <!-- Include only if your app benefits from precise location access. -->
+  <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+</manifest>
+```
+
+2. Create a new activity named `BasicLocationPulsingCircleActivity`:
+  - This Activity Implements the `OnMapReadyCallback` interface. The `onMapReady()` method is used to manage the application once the map is ready.
+  - Add a variable `permissionsManager` to manage permissions.
+  - Add a variable `locationComponent` to manage user location.
+  - At the end of the `onCreate()` method, call `checkPermissions(`) to ensure that the application can access the user's location.
+
+```kotlin
+{{#include ../../../../platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/location/BasicLocationPulsingCircleActivity.kt:top}}
+```
+
+3. In the `checkPermissions()` method, the [PermissionManager] is used to request location permissions at runtime and handle the callbacks for permission granting or rejection.
+
+   Additionally, you should pass the results of `Activity.onRequestPermissionResult()` to it.
+
+   If the permissions are granted, call `mapView.getMapAsync(this)` to execute the `onMapReady()` method when the map is ready.
+
+```kotlin
+{{#include ../../../../platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/location/BasicLocationPulsingCircleActivity.kt:permission}}
+```
+
+4. In the `onMapReady()` method, first set the style and then handle the user's location using the [LocationComponent].
+
+   To configure the [LocationComponent], developers should use [LocationComponentOptions].
+
+   In this demonstration, we create an instance of this class.[^1]
+
+   In this method:
+   - Use the annotation `@SuppressLint("MissingPermission")` to suppress warnings related to missing location access permissions.
+   - In `setStyle(),` you can utilize other public and token-free styles like [demotiles] instead of the [predefined styles].
+   - For the builder of [LocationComponentOptions], use `pulseEnabled(true)` to enable the pulse animation, which enhances awareness of the user's location.
```

Comment:
here I'd just go with:

Make sure you `activateLocationComponent` of `locationComponent` You can also set `locationComponent`'s various properties like `isLocationComponentEnabled` , `cameraMode` , etc...

Reply:
How about this? c3b5f6d

```diff
    - Use method `buildLocationComponentActivationOptions()` to set [LocationComponentActivationOptions], then activate `locatinoComp
+   - To apply options, make sure you call `activateLocationComponent()` of `locationComponent`. You can also set `locationComponent`  various properties like `isLocationComponentEnabled` , `cameraMode` , etc...
    - `CameraMode.TRACKING`[^1] means that when the user's location is updated, the camera will reposition accordingly.
```